app/javascript/components/lease_ups/LeaseUpApplicationsTableContainer.js

```diff
@@ -63,6 +62,35 @@ export const buildRowData = (application) => {
   return rowData
 }
 
+export const buildApplicationsWithLayeredValidations = (
+  listingId,
+  applications,
+  preferences,
+  setPrefMap
+) => {
+  if (!preferences) return
+  // don't need layered validation for fcfs
+  // don't need layered validation for non-veteran listings
+  // don't need layered validation for initial call
```

Comment:
Should we remove the part of the comment about not needing layered validation for initial call? Because we don't implement that check